### PR TITLE
New Library Perspective (new authoring)

### PR DESCRIPTION
**Pull request by pefernan**
This adds the Library Perspective missing deps and fixes the build.

### CODE REVIEW

**Comment by ederign** [comment]
+1 thanks buddy.

File: jbpm-form-modeler-showcase/src/main/java/org/jbpm/formModeler/client/home/HomeProducer.java [hunk 1]
```diff
@@ -56,6 +58,11 @@ public void init() {
                 () -> placeManager.goTo( PerspectiveIds.AUTHORING ),
                 PerspectiveIds.AUTHORING, ActivityResourceType.PERSPECTIVE) );
 
+        s1.addChild( ModelUtils.makeSectionEntry( "Library",
```
Comment (manstis) [inline jbpm-form-modeler-showcase/src/main/java/org/jbpm/formModeler/client/home/HomeProducer.java]: I guess we can let the i18n slip...

**Comment by manstis** [comment]
Do we need Jenkins to test this before merge?

**Comment by ederign** [comment]
@manstis how starts a build from here?

**Comment by ederign** [comment]
jenkins retest this

File: jbpm-form-modeler-showcase/src/main/java/org/jbpm/formModeler/client/home/HomeProducer.java (hunk 1 above)
Comment (pefernan) [inline jbpm-form-modeler-showcase/src/main/java/org/jbpm/formModeler/client/home/HomeProducer.java]: Well... added i18n.

**Comment by manstis** [comment]
Jenkins please retest this.

File: jbpm-form-modeler-showcase/src/main/resources/org/jbpm/formModeler/client/i18n/Constants.properties [hunk 2]
```diff
@@ -4,3 +4,11 @@ logout=Logout
 home=Home
 authoring=Authoring
 project_authoring=Project Authoring
+
+homeTitle=Welcome to the jBPM Form Modeler
+model=Model
+modelText=Create your persistable models
+design=Desing
```
Comment (psiroky) [inline jbpm-form-modeler-showcase/src/main/resources/org/jbpm/formModeler/client/i18n/Constants.properties]: Typo `Desing` -> `Design`

**Comment by psiroky** [comment]
Merging as it needed to fix the build. @pefernan could you please fix the typo and create new PR?

**Pull request merged**